src/sidebar/components/share-annotations-panel.js

```diff
@@ -110,6 +113,14 @@ function ShareAnnotationsPanel({ analytics, toastMessenger }) {
           />
         </div>
       )}
+      {shareURI && !uriShareable && (
```

Comment:
The conditions here imply that we are missing a state for when this panel is shown before the document URI is known (eg. because the document is a PDF that still loading). Currently the output is an empty panel with a title but no content. It might be a bit nicer to show a loading spinner or similar.

Reply:
I've rewritten this logic in the newest iteration.